Bottom: ab98d5e73a088b8f90ab5e4bd5eb21921c353df9
Top:    62845dfea692e5d1f0cf7a300424ebbdb91e7fea
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-16 00:57:48 +0000

Put back layer opacity to its old behaviour

The exact details of this setting aren't right yet,
so revert them so I can build a release.


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 25c4bf7..32edcf3 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1806,7 +1806,7 @@ ghid_draw_everything (BoxTypePtr drawn_area)
   }
 
   /* draw all layers in layerstack order */
-#define FADE_FACTOR 0.6
+#define FADE_FACTOR 1
   number_phys_on_top = max_phys_group - min_phys_group;
   for (i = ngroups - 1; i >= 0; i--) {
     bool is_this_physical = drawn_groups[i] >= min_phys_group &&
